packages/dev/core/src/XR/features/WebXRDepthSensing.ts

```diff
@@ -0,0 +1,320 @@
+import { RawTexture } from "../../Materials/Textures/rawTexture";
+import { WebXRFeatureName, WebXRFeaturesManager } from "../webXRFeaturesManager";
+import type { WebXRSessionManager } from "../webXRSessionManager";
+import { WebXRAbstractFeature } from "./WebXRAbstractFeature";
+import { Tools } from "../../Misc/tools";
+import { Texture } from "../../Materials/Textures/texture";
+import { Engine } from "../../Engines/engine";
+import { Observable } from "../../Misc/observable";
+
+/**
+ * Options for Depth Sensing feature
+ */
+export interface IWebXRDepthSensingOptions {
+    /**
+     *  The desired depth sensing usage for the session
+     */
+    usagePreference: XRDepthUsage[];

```

Comment:
avoid using the XR types in the module configuration - they can be the same, but this should be a babylon type. i.e.:

```
type WebXRDepthUsage = "cpu" | "gpu";
```

This way babylon doesn't change even if the WebXR specs have changes (and that's the reasoning behind doing that).

Reply:
I defined some original types to resolve this.
I'd like to ask some opinions about these namings.

```ts
export type WebXRDepthUsage = "cpu" | "gpu";
export type WebXRDepthDataFormat = "ushort" | "float";
```